docs/pieces.md

```diff
@@ -2,21 +2,16 @@
 title: "Pieces"
 ---
 
-# Pieces in A3
+# Pieces
+What is a Piece?
 
-:::  tip Note:
-<!-- I realize that for alpha 1 our audience is more familiar with Apostrophe, but this is a topic that comes up even among people who know a lot about Apostrophe, so I'd like this note to stay in. So if you have a
-concern with it, work with me to make it better, don't remove it. -Tom -->
-**"What's a piece?"**
+As experienced A2 delevelopers know, a piece type is a repository for a collection of related content-types or pieces. If you have hundreds of documents of the same type, that's a piece. If you want to let the user view them in a certain sorting order, that's a piece. If the same item might be appropriate to display on many different pages, that's a piece. Blog posts, events and products are all excellent examples. Less obvious examples include "category" or "tag" content types that serve to create a taxonomy for other pieces.
```

Comment:
typo: "delevelopers"

> a piece type is a repository for a collection of related content-types or pieces

That is less clear to me. "piece types define structure and behavior for content categories, such as blog posts, products, or events."

> If you have hundreds of documents of the same type, that's a piece.
That's not enough in and of itself, which is why combining the criteria in that sentence is useful.

It looks like the question is on the first sentence. My suggestion: "As experienced A2 developers know, piece types define structure and behavior for categories of content, such as blog posts, products, or events." We don't need to go for a full Websters Dictionary definition.

Reply:
I agree with Bea's points here. Also we need to avoid confusing people about a key point: a piece type module defines ONE content type. And, we can't lose the distinction from pages.

How about:

"As experienced A2 developers know, a piece type module defines structure and behavior for a single type of content, such as blog posts, products or events, that shouldn't be represented as a page in a tree."

And the rest of the paragraph would stay as-is, it sounds like.

Reply:
I was trying to avoid "type of content" or "content type" since it seems close to defining a word with itself. So I'd change my bit to "a piece type defines the structure and behavior for a category of content"

Reply:
The page tree thing seems irrelevant. It's true, but between examples below and devs' experience, I don't think it needs to be stated here.

Reply:
Hmm I guess it would be hard to confuse a page with anything suited to these use cases, maybe it doesn't need to be in sentence one after all. But I'd like the distinction to at least be mentioned nearby.

Reply:
Drupal, Wordpress and other CMSes use "content type" when explaining similar concepts. I think it's clear and concise.

Reply:
"Content type" is what Drupal calls the thing. Wordpress uses "post type." We have "piece type," so I want to avoid "type" in the definition in order to be a sort of definition. But not worth bike shedding now. I'll make an update for review.